nvme.c

```diff
@@ -1759,6 +1759,70 @@ static int id_ns(int argc, char **argv, struct command *cmd, struct plugin *plug
 	return err;
 }
 
+static int id_nvmset(int argc, char **argv, struct command *cmd, struct plugin *plugin)
+{
+	const char *desc = "Send an Identify NVM Set List command to the "\
+		"given device, returns entries for NVM Set identifiers greater "\
+		"than or equal to the value specified CDW11.NVMSETID "\
+		"in either binary format or json format";
+	const char *raw_binary = "show infos in binary format";
+	const char *nvmset_id = "NVM Set Identify value";
+	int err, fmt, fd;
+	struct nvme_id_nvmset nvmset;
+
+	struct config {
+		__u16 nvmset_id;
+		int   raw_binary;
+		char *output_format;
+	};
+
+	struct config cfg = {
+		.nvmset_id = 0,
+		.output_format = "normal",
+	};
+
+	const struct argconfig_commandline_options command_line_options[] = {
+		{"nvmset_id",       'i', "NUM", CFG_POSITIVE, &cfg.nvmset_id,       required_argument, nvmset_id},
+		{"raw-binary",      'b', "",    CFG_NONE,     &cfg.raw_binary,      no_argument,       raw_binary},
```

Comment:
'raw-binary' seems superfluous and breaks single point of truth. why did you introduced it?

Reply:
Why do you think it's 'superfluous' ?
I believe there might be an use case that user wants to pipe or redirect the output of this command to the other one.

Reply:
I think that '-o binary' can be sufficient to handle the same. Otherwise, it can rise questions why '-b' has higher priority. The result of '-o binary > /tmp/out.bin' and '-b > /tmp/out.bin' still the same.

Reply:
I'm not sure about what history was there about this, but many subcommands support both of them.
I was trying to follow the previous one, but I think some needs to be revised at one time if decided not to be used.
What do you say?

Reply:
I'd got with '-o', but @keithbusch  is the one to decide.

Reply:
I'd really like to deprecate the "--raw-binary" and use the output-format instead.

Reply:
Thanks for your reviews.
I have updated pull request with deprecation of --raw-binary.